docs/learn-collator.md

```diff
@@ -28,19 +28,34 @@ the censorship problem is solved with having just one honest collator.
 
 ### XCMP
 
-Collators are a key element of [XCMP (Cross-Chain Message Passing)](learn-crosschain). By being
+Collators are a key element of [XCMP (Cross-Chain Message Passing)](Due to the way the [Phragmen algorithm](learn-phragmen) generates the solution set, there _may_
+exist a minimum number of DOT needed to nominate with in order to receive staking
+rewards due to the fact that the solution set needs to be both processed and stored within
+a single block: [a target of 6 seconds on Polkadot](faq#what-is-the-block-time-of-the-relay-chain).
+
+A [runtime upgrade](build-protocol-info#runtime-upgrades) moved the logic to process the rewards
+for nominators to an [off-chain worker](learn-phragmen#off-chain-phragmén), and now the minimum
+DOT required is only limited by the [existential deposit](build-protocol-info#existential-deposit).
+If the number of nominators increases dramatically, the solution will be pruned and nominators with
+the least amount of stake will once again be pruned and receive no staking rewards to ensure that 
+block times remain on target.
+
+See the [Election Solution Set](learn-nominator#the-election-solution-set) section on the
+nominator page for more details.
+). By being
 full-nodes of the Relay Chain, they are all aware of each other as peers. This makes it possible for
 them to send messages from parachain A to parachain B.
 
 ### Collators in the Wild
 
-Collators are being used on many production and testing relay chains now!
-You can learn more about how they are used with Cumulus via the
-[Cumulus repository](https://github.com/paritytech/cumulus/).
+Collators are being used on by parachains on many
```

Comment:
Kind of pointless. Collators are used by all parachains.

Reply:
removed